packages/wallets/src/features/cashier/modules/DepositCrypto/DepositCrypto.tsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
+import { Divider } from '../../../../components/Base';
+import { TransactionStatus } from '../TransactionStatus';
 import DepositCryptoAddress from './components/DepositCryptoAddress/DepositCryptoAddress';
 import DepositCryptoCurrencyDetails from './components/DepositCryptoCurrencyDetails/DepositCryptoCurrencyDetails';
 import DepositCryptoDisclaimers from './components/DepositCryptoDisclaimers/DepositCryptoDisclaimers';
 import DepositCryptoTryFiatOnRamp from './components/DepositCryptoTryFiatOnRamp/DepositCryptoTryFiatOnRamp';
-import { Divider } from '../../../../components/Base';
 import './DepositCrypto.scss';
 
 const DepositCrypto = () => {
     return (
-        <div className='wallets-deposit-crypto'>
-            <DepositCryptoCurrencyDetails />
-            <DepositCryptoAddress />
-            <DepositCryptoDisclaimers />
-            <Divider />
-            <DepositCryptoTryFiatOnRamp />
+        <div className='wallets-deposit-crypto-container'>
```

Comment:
(minor/optional) as per BEM convention, might be better to use "deposits-crypt__container"
typcially block is the name of component, and element is separated from block by double underscore,

Reply:
Oh, missed this one, thanks